internal/storage/storage.go

```diff
@@ -358,6 +468,15 @@ func loadPackageContents(filesystem billy.Filesystem, resourcePaths []string) (f
 	return m, nil
 }
 
+func transformPackageContents(contents fileContents, transform func(string, []byte) (string, []byte)) fileContents {
```

Comment:
Consider defining a custom type like `type contentTransformer func(string, []byte) (string, []byte)` so you can use that in function definitions instead.

Reply:
Fixed!